timon/configure.py

```diff
@@ -228,6 +229,76 @@ def sort_key_func(kval):
     return ordered_cfg
 
 
+def get_data(fieldname, entry):
+    field = fieldname.split(".")
+    rslt = entry
+    for data in field:
+        if rslt[data]:
+            rslt = rslt[data]
+    return rslt
+
+
+def mk_nested_dict(grpby, entries):
```

Comment:
the function sounds generic, but is only used for grpby.
perhaps call it cnvt_grpby_to_nested_dict()

Also add a doc string

Reply:
I also think, that timon/configure.py might be getting a little big.
Perhaps factor some functions out into timon/conf/grpby.py

However if doing this don't forget to add an empty timon/conf/__init__.py and to add
a line with
"timon.conf" into the packages parameter of setup.py (or enable auto detection of packages, google would be your friend. However I suggest to do auto discovery in a separate PR)